src/App.tsx

```diff
@@ -1,61 +1,137 @@
+import React, { useState } from 'react';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { TodoList } from './components/TodoList';
+
+import usersFromServer from './api/users';
+import { User } from './types/User';
+
+import todosFromServer from './api/todos';
+import { Todo } from './types/Todo';
+
+function findUser(id: number):User | null {
+  return usersFromServer.find(user => user.id === id)
+    || null;
+}
+
+const initialTodos:Todo[] = todosFromServer.map(todo => ({
+  ...todo,
+  user: findUser(todo.userId),
+}));
 
 export const App = () => {
+  const [todos, setTodos] = useState(initialTodos);
+
+  const [newTitle, setNewTitle] = useState('');
+  const [titleError, setTitleError] = useState(false);
```

Comment:
Rename all boolean variables using [tips](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig)
```suggestion
  const [isTitleError, setIsTitleError] = useState(false);
```